contrib/build_static_lib.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+#
+# Copyright (C) Mellanox Technologies Ltd. 2001-2017.  ALL RIGHTS RESERVED.
+# See file LICENSE for terms.
+#
+
+# This scripts creates a single static UCX library, which also includes
+# ibverbs and ibcm libs.
+
+UCX="../install"
```

Comment:
i would rename these vars to UCX_PREFIX and IB_PREFIX, and force user to provide non-empty UCX prefix, instead of assuming ../install